src/app/shared/model/story.model.ts

```diff
@@ -134,9 +135,20 @@ export class StoryModel extends BaseModel {
     }
   }
 
-  get unsavedImage(): ImageModel {
-    let img = new ImageModel(this.parent, this.doc, null);
-    return img.isStored() && !img.isDestroy ? img : null;
+  addImage(upload: Upload): ImageModel {
+    let image = new ImageModel(this.doc, upload);
+    this.images.push(image);
```

Comment:
I'm finding that change detection isn't working for the wysiwyg editor insert image menu. Can addImage and removeImage set this.images to new arrays instead (both for story and series)?
`this.images = [...this.images, image];`
`this.images = [...this.images.splice(this.images.indexOf(image), 1)];`
or `slice` or whatever you like 😃